component.pl

```diff
@@ -0,0 +1,74 @@
+component(Diagram,C,Name,Ins,Outs,Children,Connections) :-
```

Comment:
All the files for your entry need to be in your team's directory

Reply:
I haven't done this before and I haven't done a forked PR before.

I need clarification, if you have time...

I have not included node_modules and /usr/local/bin/swipl.  Swipl is a PROLOG compiler (SWI Prolog).

Should I be including these in the PR?

It is not obvious to me what is missing.

thanks!
pt

Reply:
Hello guitarvydas,

In your computer move all your project files from `jam0001` to your `jam0001/guitarvydas` directory

for example, your files `component.pl` and `jsoncomponent.pl` are on the `jam0001` directory outside your team directory

Just move them and then create and push a new commit, no need to create another PR

![help](https://user-images.githubusercontent.com/23631699/130369165-100353b7-70f1-46c0-b723-d242aaaccb55.PNG)

Reply:
@guitarvydas Your `.gitignore` also should be in your project directory because it is related to it, not to all project